maven-help-plugin/src/main/java/org/apache/maven/plugins/help/EvaluateMojo.java

```diff
@@ -373,7 +373,7 @@ private void handleResponse( String expr )
 
         if ( obj != null && expr.equals( obj.toString() ) )
         {
-            getLog().warn( "The Maven expression was invalid. Please use a valid expression." );
+            System.out.println( "The Maven expression was invalid. Please use a valid expression." );
```

Comment:
As far as I know, no Maven sends output to `stderr`.  I am not going to break this convention.